src/ClientTlsContext.php

```diff
@@ -24,6 +24,12 @@ final class ClientTlsContext
 
     private int $verifyDepth = 10;
 
+    private bool $allowSelfSigned = false;
+
+    private bool $disableCompression = true;
```

Comment:
Why is this enabled by default?

Reply:
You mean disabled by default? Because that is PHP's default.

Reply:
@kelunik raised a good point in chat – compression is removed in future TLS version anyway, so there's no reason to have an option for it.